mento-protocol/mento-deployment
# feat: add minipay airdrop csv generation script

### Description

This pr adds the script for generating the minipay airdrop csv from the dune query result.

### Other changes



### Tested

Ran the script and the size of csv entries matched the size of the query rows. Verified calculation is precise.
How to Test since running on the actual query could get expensive:
1. fork the dune query and limit the result to e.g 10( add `LIMIT 10` to the bottom of the query)
2. change the queryid in the script to match your fork of the query (Line 37 `const queryId = .... ;)`
3. change the limit in line 45 of the script `const limit = 2;`
(results in 2 rows being requested from the API per request)
5. verifying the results + number of entries in the csv 

### Related issues

- Fixes #215 

### Backwards compatibility



### Documentation
